lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -561,6 +561,60 @@ ospf:
     config_get_token: ['/^router ospf <name>$/i', '/^vrf\s+(\S+)$/']
     config_set: ["router ospf <name>", "<state> vrf <vrf>", "end"]
 
+radius_server:
+  hosts:
+    config_get: "show running-config radius"
+    config_get_token: '/^radius-server host ((?:[0-9]{1,3}\.){3}[0-9]{1,3})/'
+    config_set: '<state> radius-server host <ip>'
+    default_value: ''
+
+  auth-port:
+    config_get: "show running-config radius"
```

Comment:
Consider using 'show running-config radius all' so as to display even default values of configs.